### PR TITLE
Update GLideN64.ini NVIDIA-Optimized

### PR DESCRIPTION
**Pull request by Predator82Germany**
With added Good_Name & alphabetically sorted after this

### CODE REVIEW

File: GLideN64.ini [hunk 1]
```diff
@@ -57,83 +57,153 @@ blendMode=0
...
+frameBufferEmulation\copyFromRDRAM=1
+
 [LEGORacers]
+Good_Name=Lego Racers (E)(U)
 frameBufferEmulation\detectCFB=1
 
+[STARFOX64]
+Good_Name=Lylat Wars (A)(E) / Star Fox 64 (J)(U)
+frameBufferEmulation\copyToRDRAM=0
+frameBufferEmulation\validityCheckMethod=1
```
Comment (baptiste0602) [inline GLideN64.ini]: It's not necessary since #363 is fixed

**Comment by baptiste0602** [comment]
Good work Predator.

Since the below options are default values, maybe it's not necessary to force them (what do you think @gonetz ?) : 
copyToRDRAM=1 (not in this gliden64.ini but option is check when you press "Resore Defaults")
copyDepthToRDRAM=1
N64DepthCompare=0

**Comment by gonetz** [comment]
From the one side setting custom option to its default value is redundant change.
From the other side, it can make sense if you know, that that setting is essential for correct game work. Then the game will work correct even if defaults will be changed for some reason.

File: GLideN64.ini (hunk 1 above)
Comment (Predator82Germany) [inline GLideN64.ini]: Ok thanks

**Comment by Predator82Germany** [comment]
N64DepthCompare=0 have I forced cause they normally set to 1 but NVIDIA don´t like this
So if this gets fixed anytime they can set to 1
Others are only forced cause needed & not defaul

**Comment by gonetz** [comment]
Ok, lets try to fix depth compare after finish with #374

**Pull request merged**